tests/test_nystrom_attention.py

```diff
@@ -13,14 +13,14 @@
 
 @pytest.mark.parametrize("pinverse_original_init", [True, False])
 @pytest.mark.parametrize("use_razavi_pinverse", [True, False])
-@pytest.mark.parametrize("num_landmarks", [30, 33, 905])
```

Comment:
I think it's still valuable to test this because we test different code paths, one where the sequence length is divisible by the number of landmarks and one where it isn't

Reply:
Currently I have [33, 905], would [30, 905] be preferable? This is a somewhat slow test, so I'm trying to reduce the test cases.

Reply:
Based on this feedback, I refactored this test into two separate tests. The "close_to_sdp" test includes are three parameters and their original cases, the other test has fixed parameters.

Reply:
By the way, is it intentional that the logic is repeated within the test_close_to_sdp() function?